src/components/dashboard/main-content/course-enrollments/course-cards/mark-complete-modal/ModalBody.jsx

```diff
@@ -13,11 +13,18 @@ const ModalBody = () => {
     <>
       {confirmError && <ModalError />}
       <p className="m-0">
-        Are you sure you want to mark
+        Are you sure you want to archive
         {' '}
-        <a href={courseLink}>{courseTitle}</a>
+        <a href={courseLink}>{courseTitle}</a>?
         {' '}
-        as complete?
+        You will remain enrolled, but the course will
+        no longer appear as &quot;In Progress&quot;.
+      </p>
+      <p className="mt-1">
```

Comment:
nit: perhaps `mt-2` to add a tiny bit more white space for separation between the paragraphs.